apps/mdn/mdn-aws/docs/mdn_maintenance_mode_checklist.md

```diff
@@ -0,0 +1,134 @@
+Here's some things to check when MDN is in maintenance mode.
+
+This checklist uses these URLs:
+
+* Base: https://mdn-mm.moz.works
+* CDN: https://mm-cdn.mdn.moz.works
+* Demo: https://mdn-mm-demos.moz.works
+* Interactive Examples: https://interactive-examples.mdn.moz.works
+
+# Preparing for Maintenance Mode
+
+If you want a public-facing site, you'll need to generate
+content before putting the site in Maintenance Mode. The
+full steps are in *todo*, but here's the quick version:
+
+* [ ] Load a recent (anonymized) backup
+* [ ] Enable GitHub auth (see [Enable GitHub Auth](https://kuma.readthedocs.io/en/latest/installation.html#enable-github-auth-optional))
```

Comment:
In maintenance mode, user cant log in. So Github auth is required?

Reply:
I think you need to be logged in and use the Django admin to generate the search index, so you'll need to do some things before switching to MM.

Reply:
In the anonymized databse, the email is anonymized also. So if any admin tries to log in with github, he will be unable to do.
Moreover, I think admin should able to login with username password in the `/admin` url!